src/Bridge/Yii2ComposerBridge.php

```
@@ -0,0 +1,28 @@
+ *
+ * Version 1 behavior: notice only — no automatic wiring is performed.
+ *
+ * @copyright Copyright (C) 2025 Terabytesoftw.
+ * @license https://opensource.org/license/bsd-3-clause BSD 3-Clause License.
+ */
+final class Yii2ComposerBridge
+{
+    /**
+     * Writes a one-time interop notice to the IO layer.
+     */
+    public static function logNotice(IOInterface $io): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $io. Configured minimum length is 3. (undefined)

(ShortVariable)